doc/source/tutorial/fft.rst

```diff
@@ -472,22 +480,32 @@ and normalizations.
 
 >>> from scipy.fft import dst, idst
 >>> x = np.array([1.0, 2.0, 1.0, -1.0, 1.5])
+>>> dst(dst(x, type=2, norm='ortho'), type=3, norm='ortho')
+array([ 1. ,  2. ,  1. , -1. ,  1.5])
 >>>  # scaling factor 2*N = 10
->>> idst(dst(x, type=2), type=2)
+>>> dst(dst(x, type=2), type=3)
```

Comment:
It might be work making this comment more explicit now for readers who may have skimmed past the text on the relation between DST II/III above.

something like:
```# Unnormalized round-trip via type-iii dst: scaling factor 2*N```

Reply:
I've made the first example for DCT-II more explicit and updated the comments for the rest.